features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -17,6 +17,14 @@
  */
 
 import { Show } from "@wso2is/access-control";
+import { ApplicationEditForm } from "@wso2is/admin.application-templates.v1/components/application-edit-form";
+import { ApplicationMarkdownGuide } from "@wso2is/admin.application-templates.v1/components/application-markdown-guide";
+import useApplicationTemplateMetadata from
+    "@wso2is/admin.application-templates.v1/hooks/use-application-template-metadata";
```

Comment:
```suggestion
import useApplicationTemplateMetadata from "@wso2is/admin.application-templates.v1/hooks/use-application-template-metadata";
```

Reply:
We can't add an import like this; it results in an error.

Reply:
Ah, I see. Okay then